lib/srv/sshserver_test.go

```diff
@@ -408,41 +408,6 @@ func (s *SrvSuite) TestExecCommandLocalPtyAndNoRemotePty(c *C) {
 	c.Assert(string(output), Equals, "1\r\n2\r\n")
 }
 
-// TestExecCommandNoLocalAndPtyRemotePty tests executing a command where you
-// have no local PTY but have a remote PTY.
-func (s *SrvSuite) TestExecCommandNoLocalPtyAndRemotePty(c *C) {
-	// 1 - command: seq 2 | ssh -tt localhost 'stty -opost; echo'
```

Comment:
instead of removing, you just `c.Skip()` and add warning, because we want this back